indra/newview/llfloateremojipicker.cpp

```diff
@@ -139,7 +155,7 @@ void LLFloaterEmojiPicker::fillEmojis()
 	mEmojis->clearRows();
 
 	const LLEmojiDictionary::emoji2descr_map_t& emoji2Descr = LLEmojiDictionary::instance().getEmoji2Descr();
-	for (const LLEmojiDictionary::emoji2descr_item_t& it : emoji2Descr)
```

Comment:
Why is it not a reference?

Reply:
Because using a reference causes a compilation error on macOS